x/stakeibc/client/cli/query_lsm_deposits.go

```diff
@@ -0,0 +1,107 @@
+package cli
+
+import (
+	"context"
+	"errors"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/spf13/cobra"
+
+	"github.com/Stride-Labs/stride/v8/x/stakeibc/types"
+)
+
+func CmdLSMDepositsHostZone() *cobra.Command {
```

Comment:
I think we should just have one command with optional flags. So the command is just `lsm-deposits` which defaults to all deposits. And then there's optional `--host-zone`, `--denom`, and `--status` flags

Here's an example of how to add the flags [[this](https://github.com/Stride-Labs/stride/pull/619/files#diff-0087d5d0ca1a2154aadad78b1582fa52f24d66f8583a0e08ba76c63be6d4adadR81) function, [flag](https://github.com/Stride-Labs/stride/pull/619/files#diff-0087d5d0ca1a2154aadad78b1582fa52f24d66f8583a0e08ba76c63be6d4adadR133) and [flag](https://github.com/Stride-Labs/stride/pull/619/files#diff-0087d5d0ca1a2154aadad78b1582fa52f24d66f8583a0e08ba76c63be6d4adadR95)]